### PR TITLE
Dependency updates for Angular 8 and TS 3.3.3333

### LINKED ISSUE #13842: `const enum` breaks when running production build/serve
# 🐞 Bug report

### Command (mark with an `x`)

```
- [ ] new
- [x] build
- [x] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Is this a regression?

Not sure about this. This seems to be a reappearance of #9571.

### Description

For a `const enum` of type `number`, like `const enum TestEnum { DESC = 2, }`, usage of this kind `TestEnum.DESC.toString(10)` breaks on `ng serve/build --prod`.

## 🔬 Minimal Reproduction

My global environments:
`node --version // v11.10.1`
`npm --version // 6.9.0`
`npm list -g --depth=0 | grep angular // @angular/cli@7.3.5`

1. On Desktop, run `ng new prod-build-test-proj`;
1.1 ? Would you like to add Angular routing? No
1.2 ? Which stylesheet format would you like to use? CSS    (.css )
2. `cd prod-build-test-proj/`;
3. Add following code to the bottom of file `src/app/app.component.ts`:
```
const enum TestBuildProd {
  OPT1 = 1,
  OPT2 = 2,
}

console.log(TestBuildProd.OPT1.toString(10));
```
4. Run `npx ng build`, success;
5. Run `npx ng build --prod`, fail;

## 🔥 Exception or Error
<pre><code>

ERROR in ./src/app/app.component.ts 8:25
Module parse failed: Unexpected token (8:25)
You may need an appropriate loader to handle this file type.
| }());
| export { AppComponent };
> console.log(1 /* OPT1 */..toString(10));
| 
</code></pre>

## 🌍 Your Environment
<pre><code>

Angular CLI: 7.3.5
Node: 11.10.1
OS: darwin x64
Angular: 
... 

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.13.5
@angular-devkit/core         7.3.5
@angular-devkit/schematics   7.3.5
@schematics/angular          7.3.5
@schematics/update           0.13.5
rxjs                         6.3.3
typescript                   3.2.4
</code></pre>

**Anything else relevant?**

If the code was appended to `main.ts` then everything works normally, this error only occurs when it was appended to `*.service.ts` or `*.component.ts`;

### PR DESCRIPTION
**Pull request by filipesilva**
Fix #13842

### CODE REVIEW

**Comment by alan-agius4** [comment]
We need to wait, until the framework supports Ts 3.3

PR for this https://github.com/angular/angular/pull/28497

File: packages/ngtools/webpack/package.json [hunk 1]
```diff
@@ -28,14 +28,14 @@
     "webpack-sources": "1.3.0"
   },
   "peerDependencies": {
-    "@angular/compiler-cli": ">=5.0.0 <8.0.0 || ^7.0.0-beta.0",
-    "typescript": ">=2.4.0 < 3.3",
+    "@angular/compiler-cli": ">=5.0.0 <9.0.0 || ^7.0.0-beta.0 || ^8.0.0-beta.0",
+    "typescript": ">=2.4.0 < 3.4",
```
Comment (alan-agius4) [inline packages/ngtools/webpack/package.json]: Should we continue supporting so many versions of typescript without proper testing?

As a side note, framework dropped support for TypeScript versions prior to 3.3 in Version 8.

Comment (filipesilva) [reply packages/ngtools/webpack/package.json]: That's a good question. 

@hansl I think you checked what the supporter Angular versions were for CLI v8, can you advise?

File: packages/schematics/angular/utility/latest-versions.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 export const latestVersions = {
   // These versions should be kept up to date with latest Angular peer dependencies.
-  Angular: '~8.0.0-beta.5',
+  Angular: '~8.0.0-beta.6',
   RxJs: '~6.4.0',
   ZoneJs: '~0.8.26',
   TypeScript: '~3.2.2',
```
Comment (alan-agius4) [inline packages/schematics/angular/utility/latest-versions.ts]: This should also be updated. As we should scaffold projects using TypeScript 3.3

File: packages/ngtools/webpack/package.json (hunk 1 above)
Comment (alan-agius4) [reply packages/ngtools/webpack/package.json]: @filipesilva I was referring to supported TypeScript versions, but yeah same thing applies for supported Angular versions.

Comment (filipesilva) [reply packages/ngtools/webpack/package.json]: They are linked together really. We only support older TS because of Angular.

Comment (clydin) [reply packages/ngtools/webpack/package.json]: Definitely one version down.  Upwards was the open question.

**Comment by alan-agius4** [comment]
@filipesilva updating to TS3, would also fix this issue https://github.com/angular/angular-cli/issues/13842

File: packages/ngtools/webpack/package.json (hunk 1 above)
Comment (alan-agius4) [reply packages/ngtools/webpack/package.json]: I think we should update this as per @alexeagle PR https://github.com/angular/angular-cli/pull/13910.
```
"@angular/compiler-cli": ">=6.0.0 <9.0.0 || ^8.0.0-beta.0",
 "typescript": ">=2.7 < 4",
```

Comment (filipesilva) [reply packages/ngtools/webpack/package.json]: TypeScript does not follow semver so using `<4` opens us up to breaking changes. Our policy is to move lockstep with their minor. Is there a reason to use `<4` this time that I am unaware of?

Comment (alan-agius4) [reply packages/ngtools/webpack/package.json]: @filipesilva, apologies it should be;

```
 "typescript": ">=2.7 < 3.4",
```

File: yarn.lock [hunk 3]
```diff
@@ -2,33 +2,33 @@
 # yarn lockfile v1
 
 
-"@angular/animations@^8.0.0-beta.0":
-  version "8.0.0-beta.3"
-  resolved "https://registry.yarnpkg.com/@angular/animations/-/animations-8.0.0-beta.3.tgz#51d41b7b800f569a82f77e6f54f0a74c85b7e8d7"
-  integrity sha512-DpeTE/HVZxsem8IkMjxNPIi71+juKlCwmZX+IsD+txR3o9xj8XdDXEliBdC1bulA8/6rFqfmwbvru0wonX+UCA==
+"@angular/animations@^8.0.0-beta.6":
```
Comment (alan-agius4) [inline yarn.lock]: The lock file seems to be outdated.

**Comment by alan-agius4** [comment]
@filipesilva what you mind having a look at this? I cannot add you as a reviewers since you authored this.

**Pull request merged**